templates/test/by_id.html

```diff
@@ -67,7 +67,34 @@
                         </tbody>
                     </table>
                 {% else %}
-                    <p>This test is still in the queue! Please check again later.</p>
+                      <p>This test is still in the queue! Please check again later.Wait {{time}} minutes</p>
+                      <p>Other Tests in queue</p>
```

Comment:
There's no need to show the user this list.... It's already available on the index page. Just letting the user know there are still x tests queued before this one is sufficient.